app/assets/javascripts/compute_resource.js

```diff
@@ -248,7 +248,7 @@ function vsphereGetResourcePools(item) {
   var data = {cluster_id: $(item).val()};
   var url = $(item).data('url');
   tfm.tools.showSpinner();
-  selectbox = $('*[id*=resource_pool]');
+  selectbox = $('*[id=host_compute_attributes_resource_pool]');
```

Comment:
I'm not sure why using a more specific selector would cause the spinner to hide? 
Any select box selected by this will have also been selected by the previous one, and I suspect there was a reason for using a more generalized selector before - perhaps this code is also used on other pages where the select box has a different ID? Even if not, matching on a precise id should be done with `$(#id)`, not `$(*[id=id])`.